internal/event/event.go

```diff
@@ -34,11 +34,58 @@ type Event struct {
 }
 
 const (
-	TypeState           = "state"
-	TypeAcknowledgement = "acknowledgement"
-	TypeInternal        = "internal"
+	TypeState                  = "state"
+	TypeAcknowledgementSet     = "acknowledgement-set"
+	TypeAcknowledgementCleared = "acknowledgement-cleared"
+	TypeInternal               = "internal"
+	TypeDowntimeRemoved        = "downtime-removed"
+	TypeDowntimeStarted        = "downtime-started"
+	TypeDowntimeTriggered      = "downtime-triggered"
```

Comment:
started vs. triggered? There's no such differentiation within the [Icinga 2 notification config](https://icinga.com/docs/icinga-2/latest/doc/09-object-types/#notification). For me, that's just a weird implementation detail within Icinga 2 that happens to be exposed via the event stream, but I'd try to avoid further exposing both.